### PR TITLE
Add exed

### LINKED ISSUE #19383: Add Exed instrument
Add the Exed instrument from HZB to Mantid

### PR DESCRIPTION
**Pull request by granrothge**
Description of work.

**To test:**

Open DGSPlanner from MantidPlot.
Pick Exed, 
Select plot
 

Fixes #19383.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant documentation been added/updated?
- [ ] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by AndreiSavici — CHANGES_REQUESTED** [review]
Pretty important milestone to add a new facility. I think it should be added to release notes as well. Maybe even a picture for the release notes

**Comment by AndreiSavici** [comment]
When testing this PR, I found a bug introduced since last release: I cannot do the fast plotting (masking detectors). I get 
```
MaskDetectors-[Notice] MaskDetectors started
MaskDetectors-[Error] Error in execution of algorithm MaskDetectors:
MaskDetectors-[Error] _Map_base::at
```

File: scripts/DGSPlanner/InstrumentSetupWidget.py [hunk 1]
```diff
@@ -150,7 +150,7 @@ def __init__(self,parent=None):
         metrics=QtGui.QFontMetrics(self.font())
         self.signaldict=dict()
         #instrument selector
-        self.instrumentList=['ARCS','CNCS','DNS','FOCUS','HET','HYSPEC','LET','MAPS','MARI','MERLIN','SEQUOIA']
+        self.instrumentList=['ARCS','CNCS','DNS','FOCUS','HET','HYSPEC','LET','MAPS','MARI','MERLIN','SEQUOIA','EXED']
```
Comment (granrothge) [inline scripts/DGSPlanner/InstrumentSetupWidget.py]: I'll Fix these.

File: instrument/EXED_Definition.xml [hunk 2]
```diff
@@ -0,0 +1,590 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<instrument name="EXED" date="6th Mar 2017"
```
Comment (granrothge) [inline instrument/EXED_Definition.xml]: I will probably pull the date field.  I'll look at the instrument definition and make sure it is more in line with standards.

**Comment by granrothge** [comment]
I am working on a load routine.  Once that is ready for a pull request,  I will add the instrument to the release notes.

File: instrument/EXED_Definition.xml (hunk 2 above)
Comment (granrothge) [inline instrument/EXED_Definition.xml]: I matched the date format to other instrument definition files and updated the pixel sizes appropriately

**Review by AndreiSavici — APPROVED** [review]

**Comment by AndreiSavici** [comment]
Looks good. :shipit:

**Comment by AndreiSavici** [comment]
The `ValidateInstrumentDefinitionFiles` systemtest failed
```
22:17:40 Validating '/opt/mantidunstable/instrument/EXED_Definition.xml'
22:17:40 VALIDATION OF '/opt/mantidunstable/instrument/EXED_Definition.xml' FAILED WITH ERROR:
22:17:40 ERROR: EXED_Definition.xml: line 2: Used root tag (None, u'instrument') not found in schema file(s)!
```

**Comment by granrothge** [comment]
I think I fixed the schema problem.  We'll see.

**Comment by granrothge** [comment]
Finally passed.  Can someone merge this?

**Pull request merged**